### ISSUE TITLE
Possible data type incoherence

### ISSUE THREAD

**Issue by mfalda** [plantbreeding/IPK-BrAPI-Validator#27]
I have spotted what seems to me a possible data type incoherence, since Brava requires the data type **string** for _X_ and _Y_ properties in the call "/studies/{studyDbId}/observationunits", but this is in contradiction with the (more plausible) data type **int** found for instance in the call "/studies/{studyDbId}/observationunits". The error follows:

```
 /studies/{studyDbId}/observationunits (schema mismatch)
140ms
Test: Status code should be 200
Test: ContentType is application/json
Test: Json matches schema: /schemas/v1.1/metadata.json
Test: Json matches schema: /schemas/v1.1/studies/studyDbId/observationunits.json

Response structure doesn't match schema.

Schema: /schemas/v1.1/studies/studyDbId/observationunits.json

Validation Errors: (only showing top 10)

Error: validation - type
In element: /result/data/0/X
Expected: null, string
Found: integer
Message: instance type (integer) does not match any allowed primitive type (allowed: ["null","string"])

Error: validation - type
In element: /result/data/0/Y
Expected: null, string
Found: integer
Message: instance type (integer) does not match any allowed primitive type (allowed: ["null","string"])


```

**Comment by mfalda** [comment]
I am sorry, the intended more coherent call is /studies/{studyDbId}/layout

**Comment by langeipk** [comment]
so the issue can be closed?

**Comment by mfalda** [comment]
My comment was a bit ambiguous: In the original comment I specified twice the same call "/studies/{studyDbId}/observationunits" both for the wrong and for the right call. I intended that the correct report should have been:

I have spotted what seems to me a possible data type incoherence, since Brava requires the data type string for X and Y properties in the call "/studies/{studyDbId}/observationunits", but this is in contradiction with the (more plausible) data type int found for instance in the call "/studies/{studyDbId}/layout". 

Thank you.
